src/test/java/org/llorllale/cactoos/matchers/IsTrueTest.java

```diff
@@ -48,10 +48,13 @@ void matchPositive() {
         ).affirm();
     }
 
+    /**
+     * Give the negative testing result for the invalid arguments.
+     */
```

Comment:
@baudoliver7 why add this comment here? I find it very confusing and not providing any useful information :)

Reply:
@victornoel You are right. Let's remove it.